packages/react-composites/src/composites/CallComposite/ConfigurationScreen.tsx

```diff
@@ -13,54 +13,71 @@ import { getCallingSelector } from '@internal/calling-component-bindings';
 import { Stack } from '@fluentui/react';
 import { LocalPreview } from './LocalPreview';
 import {
-  configurationStackTokens,
-  configurationContainer,
+  configurationStackTokensDesktop,
+  configurationStackTokensMobile,
+  configurationContainerStyleDesktop,
+  configurationContainerStyleMobile,
   selectionContainerStyle,
-  titleContainerStyle
+  startCallButtonContainerStyleDesktop,
+  startCallButtonContainerStyleMobile,
+  startCallButtonStyleMobile,
+  titleContainerStyleDesktop,
+  titleContainerStyleMobile
 } from './styles/CallConfiguration.styles';
 import { useLocale } from '../localization';
 
 /**
  * @private
  */
 export interface ConfigurationScreenProps {
+  mobileView?: boolean;
```

Comment:
nit: For internal interfaces, I'd prefer all options to be mandatory. And the default values should be set at the top level.

This avoids us sprinkling the code base with default values. Imagine if we wanted the default value to be `true` instead. How many files would need to change? (Or, even if we can't really change the default, it increases the chances of inconsistent default value in the implementation)

Reply:
I agree, also you could easily forget to set an import optional property, will update